dev-python/awkward-cpp/awkward-cpp-28.ebuild

```
@@ -0,0 +1,25 @@
+EAPI=8
```

```suggestion
EAPI=8

```

Another nit, but unwritten convention states that there is an empty line after `EAPI`